go/protocol/keybase1/metadata.go

```diff
@@ -271,6 +271,10 @@ type GetMerkleNodeArg struct {
 	Hash string `codec:"hash" json:"hash"`
 }
 
+type SetImplicitTeamModeForTestArg struct {
+	ImplicitTeamMode string `codec:"implicitTeamMode" json:"implicitTeamMode"`
```

Comment:
Should the mode be an enum instead? Not a big deal obviously since it's just for testing.

Reply:
I was just trying to match it with the environment variable behavior, so we wouldn't get confused when comparing the tests to the dockerfile.  But if you think it'd be better as a numeric enum I don't mind changing it.

Reply:
(I think this way makes the mdserver code a little cleaner, as I don't have to reconcile two separate enums together.)

Reply:
Yeah fair enough. Can just leave it as a string since it won't ever be called in production clients.